components/sidebar/sidebar_category/index.ts

```diff
@@ -4,24 +4,28 @@
 import {connect} from 'react-redux';
 import {bindActionCreators, Dispatch} from 'redux';
 
-import {GlobalState} from 'mattermost-redux/types/store';
 import {GenericAction} from 'mattermost-redux/types/actions';
 
+import {setCategoryCollapsed} from 'actions/views/channel_sidebar';
+import {isCategoryCollapsed} from 'selectors/views/channel_sidebar';
+import {GlobalState} from 'types/store';
+
 import SidebarCategory from './sidebar_category';
 
-function mockSetCollapsedState(categoryId: string, isCollapsed: boolean) {
-    return (dispatch: any, getState: any) => {};
+type OwnProps = {
+    category: any;
```

Comment:
This can use the `Category` type from `mattermost-redux/types/channel_categories`